0.Data_Preprocessing/scripts/0.generate_toy_data.py

```diff
@@ -0,0 +1,70 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.0
+#   kernelspec:
+#     display_name: base
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+# %%
+# Define paths
+test_data_output_path = pathlib.Path("../data/test_data_output.csv")
+
+# %%
+# set seed
+np.random.seed(40)
+# generate fake image data with 6 columns and 100 rows with image names as one of the columns and data from random gaussian data, geometric, poisson, binomial, exponential, uniform distributions
+df = pd.DataFrame(np.random.randn(100, 6), columns=list("ABCDEF"))
+# df['image_name'] = ['image_' + str(i) for i in range(100)]
+# df['geometric'] = np.random.geometric(p=0.5, size=100)
+# df['poisson'] = np.random.poisson(lam=1.0, size=100)
+# df['binomial'] = np.random.binomial(n=10, p=0.5, size=100)
+# df['exponential'] = np.random.exponential(scale=1.0, size=100)
+
+# add image name column to the dataframe
+df["image_name"] = ["image_" + str(i) for i in range(100)]
+
+df.to_csv(test_data_output_path, index=False)
```

Comment:
Probably, but this is just reproducible toy data to test things on, so a new index when loading this csv isn't bad?

Reply:
@MikeLippincott thanks, I think this isn't a big deal for now, but it could be something to think about when it comes to data testing or future functionality. In cases where someone might have accidentally included the index (as a default when using `pd.to_csv()`) could it negatively effect the outcomes?